src/Network/NWBrowser.cs

```diff
@@ -0,0 +1,152 @@
+//
+// NWBrowser.cs: Bindings the Network nw_browser_t API.
+//
+// Authors:
+//   Manuel de la Pena (mandel@microsoft.com)
+//
+// Copyrigh 2019 Microsoft Inc
+//
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.CompilerServices;
+using ObjCRuntime;
+using Foundation;
+using CoreFoundation;
+
+using OS_nw_browser=System.IntPtr;
+using OS_nw_browse_descriptor=System.IntPtr;
+using OS_nw_parameters=System.IntPtr;
+using dispatch_queue_t =System.IntPtr;
```

Comment:
Is this like using `#define` in c++? also, maybe an extra space here?

Reply:
Yes, `using` is like `#define`/macros in c++.

But why are we mapping all of these strings to `System.IntPtr`?

Reply:
Yes, I'm defining the dispatch_queue_t, OS_nw_parameters etc to be IntPtrs, so that when you read the PInvokes it makes more sense, is nicer to have: 

```csharp
static extern void nw_browser_set_queue (OS_nw_browser browser, dispatch_queue_t queue);
```
than
```csharp
static extern void nw_browser_set_queue (IntPtr browser, IntPtr queue);
```